scalyr_agent/configuration.py

```diff
@@ -1109,13 +1125,30 @@ def __verify_main_config_and_apply_defaults(self, config, file_path, apply_defau
         self.__verify_or_set_optional_int(config, 'k8s_cache_start_fuzz_secs', 0, description, apply_defaults, env_aware=True)
         self.__verify_or_set_optional_int(config, 'k8s_cache_purge_secs', 300, description, apply_defaults, env_aware=True)
 
-        # Whether to log api responses to disk (warning: this should be turned on only for debugging as it will
-        # fill up local disk over time
+        # Whether to log api responses to agent_debug.log
+        self.__verify_or_set_optional_bool(
+            config, 'k8s_log_api_responses', False, description, apply_defaults, env_aware=True
+        )
+
+        # If set to True, do not log successes (response code 2xx)
         self.__verify_or_set_optional_bool(
-            config, 'k8s_log_api_responses_to_disk', False, description, apply_defaults, env_aware=True
+            config, 'k8s_log_api_exclude_200s', False, description, apply_defaults, env_aware=True
         )
+        # Minimum response length of api responses to be logged.  Responses smaller than this limit are not logged.
+        self.__verify_or_set_optional_int(
+            config, 'k8s_log_api_min_response_len', 0, description, apply_defaults, env_aware=True
+        )
+        # Minimum latency of responses to be logged.  Responses faster than this limit are not logged.
+        self.__verify_or_set_optional_float(
+            config, 'k8s_log_api_min_latency', 0.0, description, apply_defaults, env_aware=True
+        )
+        # If positive, api calls with the same path will be rate-limited to a message every interval seconds
+        self.__verify_or_set_optional_int(
+            config, 'k8s_log_api_ratelimit_interval', 0, description, apply_defaults, env_aware=True
+        )
+
+        # TODO-163 : make other settings more aggressive
```

Comment:
Presumably this is coming in another commit?

Reply:
Yep, a different PR